Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs/Program.cs

```diff
@@ -546,6 +543,18 @@ private static List<Resource> GetRootNodes(AdjacencyGraph<Resource, Edge<Resourc
             return subgraph;
         }
 
+        private static Resource AddAndGetVertex(AdjacencyGraph<Resource, Edge<Resource>> resourceGraph, string claimName)
```

Comment:
Let's follow .NET naming conventions on this one... like ConcurrentDictionary's [GetOrAdd](https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2.getoradd?view=net-5.0) method since it has the same semantics.

Reply:
Thank you; done!